docs/plugins/trello-github-integ_plugin.md

```diff
@@ -1,23 +1,10 @@
 ## 1 `trello-github-integ` Plugin
 
-This plugin creates a new Trello board and integrate it with your GitHub repo.
+This plugin creates a new GitHub Actions workflow(trello-github-integration) and uploads it to your GitHub repo.
 
 ## 2 Usage:
 
-_This plugin depends on the following two environment variables:_
-
-- TRELLO_API_KEY
-- TRELLO_TOKEN
-
-Set the values accordingly before using this plugin.
-
-If you don't know how to create these two tokens, check out:
-- [Generate a Trello API key and token](https://docs.servicenow.com/bundle/quebec-it-asset-management/page/product/software-asset-management2/task/generate-trello-apikey-token.html)
-
-## 3 Tips:
-_Trello board description is managed by DevStream, please don't modify it._
-
-To create a Trello API key and token, see [here](https://docs.servicenow.com/bundle/quebec-it-asset-management/page/product/software-asset-management2/task/generate-trello-apikey-token.html).
+_This plugin depends on the plugin `trello`:_
```

Comment:
LIne 20: `dependsOn: [ "TOOL1_NAME.TOOL1_KIND", "TOOL2_NAME.TOOL2_KIND" ]`

Maybe this line could be updated to:

`dependsOn: [ "TRELLO_INSTANCE_NAME.trello" ]`

Reply:
Good point. Because trello-github-integ plugin can only depend on Trello plugin.